client/state/purchases/selectors/get-included-domain-transfer.ts

```diff
@@ -0,0 +1,31 @@
+import { isDomainTransfer } from '@automattic/calypso-products';
+import { isSubscription } from 'calypso/lib/purchases';
+import { getSitePurchases } from './get-site-purchases';
+import type { Purchase } from 'calypso/lib/purchases/types';
+import type { AppState } from 'calypso/types';
+import 'calypso/state/purchases/init';
+
+/**
+ * Returns a purchase object that corresponds to that subscription's transferred domain
+ *
+ * Even if a domain transfer was purchased with the subscription, it will
+ * not be returned if the domain transfer product was paid for separately (eg: if it was
+ * renewed on its own).
```

Comment:
I don't think a domain transfer can be renewed (so the parenthetical remark here should probably be renewed) but a bigger question is that I'm not sure anything in the code below enforces what this paragraph is saying?  So maybe either the code should change or the comment should change.  See below.

Reply:
I'll change the comment. I adapted it from [this method](https://github.com/Automattic/wp-calypso/blob/26e4a0e4ed9107ea195ac1395f873e1e00959d5a/client/state/purchases/selectors/get-included-domain-purchase.ts) and I did not look too much into it.

Reply:
Fixed in [ff339ae](https://github.com/Automattic/wp-calypso/pull/102997/commits/ff339aec1b0ffef7d911a4f44fcca2c28ed5f398)